### PR TITLE
Add base repository files

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: .github/workflows/lint.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    runs-on: ubuntu-latest
+    outputs:
+      addons: ${{ steps.addons.outputs.addons_list }}
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v4.1.6
+
+      - name: Find add-on directories
+        id: addons
+        uses: home-assistant/actions/helpers/find-addons@master
```
[the remark this answers is not in the document]
Comment (ludeeus) [reply .github/workflows/lint.yaml:25]: Generally yes, but not here.

File: .github/workflows/lint.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    needs: find
+    strategy:
+      matrix:
+        path: ${{ fromJson(needs.find.outputs.addons) }}
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v4.1.6
+
+      - name: Run Home Assistant Add-on Lint
+        uses: frenck/action-addon-linter@v2.15.1
```
[the remark this answers is not in the document]
Comment (ludeeus) [reply .github/workflows/lint.yaml:39]: This should fail fast and hard.

File: repository.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+name: HACS Addons repository
```
Comment (frenck) [inline repository.yaml:1]:
```suggestion
name: HACS Add-ons Repository
```

**Review by frenck — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,3 @@
-# addons
\ No newline at end of file
+# HACS add-ons
```
Comment (ludeeus) [inline README.md:1]:
```suggestion
# HACS Add-ons Repository
```

**Comment by ludeeus** [comment]
Thanks @frenck 🥇

**Pull request merged**